components/match2/commons/get_match_group_copy_paste_wiki_base.lua

```diff
@@ -12,9 +12,11 @@ Base-WikiSpecific Code for MatchList and Bracket Code Generators
 
 adjust this to fit the needs of your wiki^^
 
-]]--
+]]
+--
 
-local wikiCopyPaste = {}
+---@class MatchCopyPaste
```

Comment:
there is `Module:GetMatchGroupCopyPaste` and `module:GetMatchGroupCopyPaste`
imo should annotate both together
also we already have some anno suggestion for this in #3692